client/src/lib/slate-pointers/PointerImportNode.tsx

```diff
@@ -282,7 +317,11 @@ class PointerImportNodePresentational extends React.Component<any, any> {
         transition: "background-color 0.4s",
         whiteSpace: "nowrap",
         ":hover": {
-          backgroundColor: isLazyPointer
+          backgroundColor: isKnownAsMaliciousAnswerCandidate
+            ? "red"
+            : isKnownAsHonestAnswerCandidate
+            ? "green"
+            : isLazyPointer
             ? "red"
```

Comment:
Noticing the same `red` is being used for `isKnownAsMaliciousAnswerCandidate` and `isLazyPointer` here (and above). I think its good practice to use different colors for each state if possible.

@brachbach do you have any color preferences here?

Reply:
We don't really use lazy pointers at the moment, and the context would presumably disambiguate for the user. But curious what @brachbach thinks.